db/migrate/20230224185802_create_ingredients.rb

```diff
@@ -0,0 +1,19 @@
+class CreateIngredients < ActiveRecord::Migration[7.0]
+  def change
+    create_table :ingredients do |t|
+      t.string :name
+      t.timestamps
+      t.string :type
+      t.boolean :is_alcoholic
+      t.date :vintage
+      t.string :size
+      t.float :abv
+      t.string :purchase_url
+      t.blob :image
```

Comment:
```suggestion
```
We can remove this line. Images are actually done in the model file. They do not need to be set up in a migration.

Reply:
I'm setting up what we need for images in my PR, so I can help with this later.